gobblin-data-management/build.gradle

```diff
@@ -45,7 +45,7 @@ dependencies {
   compile externalDependency.junit
   compile externalDependency.jacksonMapperAsl
 
-  testCompile(group: 'org.apache.iceberg', name: 'iceberg-hive-metastore', version: '0.11.1', classifier: 'tests') {
+  testCompile(group: 'org.apache.iceberg', name: 'iceberg-hive-metastore', version: '1.2.0', classifier: 'tests') {
```

Comment:
We should define the dependency in gradle/scripts/dependencyDefinitions.gradle

Reply:
@arjun4084346 I just modified the existing dependency from an older version. I can define it under `gradle/scripts/dependencyDefinitions.gradle`. I wasn't sure why it was defined this way earlier